spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -87,11 +132,14 @@ def nav(self, nav_target: str = None) -> Tuple[bool, str]:
             bool: True if navigation was successful, False otherwise
             str: Message indicating the status of the navigation
         """
+        print(f"Received nav target request for - {nav_target}")
```

Comment:
Can we add a verbose flag here and put print statements under that?

Reply:
Or maybe add a global "verbose" flag to the SkillManager constructor, then every fn can print or not print based on requirement.

Reply:
Added a new method `_conditional_print(msg)` which prints based on the verbose flag